src/Moryx/Serialization/EntryConvert/EntryConvert.cs

```diff
@@ -102,6 +102,7 @@ public static Entry EncodeProperty(PropertyInfo property, ICustomSerialization c
         /// <see cref="ICustomSerialization"/>
         private static EntryValue CreateEntryValue(PropertyInfo property, ICustomSerialization customSerialization)
         {
+           
```

Comment:
sSnce you didn't really change anything in this file, can you please revert the changes